control-plane/roles/logging/templates/loki/promtail_values.yaml

```diff
@@ -0,0 +1,37 @@
+image:
```

Comment:
Also don't see where this is used. :(

Reply:
https://github.com/metal-stack/metal-roles/commit/e2d00bd91b4987c4bab6563643dfc981f8efd5a1